lib/read/lib/Cardano/Wallet/Read/Block/SlotNo.hs

```diff
@@ -62,20 +65,18 @@ getEraSlotNo = case theEra @era of
     k = SlotNo . fromIntegral . O.unSlotNo
 
 newtype SlotNo = SlotNo {unSlotNo :: Natural}
-    deriving (Eq, Show)
+    deriving (Eq, Ord, Show, Generic)
+
+instance NoThunks SlotNo
```

Comment:
This could be shortened to:
```suggestion
    deriving stock (Eq, Ord, Show, Generic)                                                                                                                                         
    deriving anyclass NoThunks   
```

Reply:
… but this is only shorter if you don't count the lines

```hs
{-# LANGUAGE DerivingStrategies #-}
{-# LANGUAGE DeriveAnyClass #-}
```

that would have to be added at the top of the file. 🤓 

I prefer to keep this code as is, as the intent is clear. In general, I only support new language extensions 
* if they provide value with a good value/cost ratio
* and if they are established as part of the "team dialect of Haskell". (mental load)